content/tutorials/grafana-fundamentals.md

```diff
@@ -71,7 +71,7 @@ In this step, you'll set up the sample application, as well as supporting servic
    docker-compose ps
    ```
 
-   In the **State** column, it should say `Up` for all services.
+   In the **STATUS** column, it should say `running` for all services.
```

Comment:
https://docs.docker.com/engine/reference/commandline/compose_ps/#options

Reply:
I just ran `docker-compose ps` and have the following columns:
```console
$ docker-compose ps
    Name                 Command             State    Ports
```

Is this change correct?

```console
$ docker-compose version

docker-compose version 1.29.2, build unknown
docker-py version: 5.0.3
CPython version: 3.9.10
OpenSSL version: OpenSSL 1.1.1n  15 Mar 2022
```